### PR TITLE
docs: add Java docs and change code style to Google Code Style

### PR DESCRIPTION
**Pull request by arvgord**
(#13)

- Changed code style
- Added Java docs in changed classes

### CODE REVIEW

File: src/main/java/com/quantumsoft/qupathcloud/entities/Datasets.java [hunk 1]
```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Datasets class contains a list of {@link com.quantumsoft.qupathcloud.entities.Dataset}
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/entities/Datasets.java]: Datasets is the list of {@link com.quantumsoft.qupathcloud.entities.Dataset} retrieved from the server. ?

File: src/main/java/com/quantumsoft/qupathcloud/entities/DicomAttribute.java [hunk 2]
```diff
@@ -17,55 +17,94 @@
...
-
 import java.util.Arrays;
 import java.util.Objects;
 
-public class DicomAttribute<T>{
-    private String vr;
-    @JsonProperty
-    private T[] Value;
+/**
+ * DicomAttribute class may contain any DICOM attribute.
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/entities/DicomAttribute.java:24]: I would describe what a DICOM Attribute  or what the supported attibutes are..this is not adding any information that will help the client.

File: src/main/java/com/quantumsoft/qupathcloud/entities/DicomStores.java [hunk 3]
```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The DicomStores class contains a list of {@link com.quantumsoft.qupathcloud.entities.DicomStore}
+ * retrieving from the server.
+ *
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/entities/DicomStores.java:24]: DicomStores contains a list of {@link com.quantumsoft.qupathcloud.entities.DicomStore}
 retrieved from the server.

File: src/main/java/com/quantumsoft/qupathcloud/entities/Locations.java [hunk 4]
```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Locations class contains a list of {@link com.quantumsoft.qupathcloud.entities.Location}
+ * retrieving from the server.
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/entities/Locations.java]: Locations contains a list of {@link com.quantumsoft.qupathcloud.entities.Location}
 retrieved from the server.

File: src/main/java/com/quantumsoft/qupathcloud/entities/Projects.java [hunk 5]
```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Projects class contains a list of {@link com.quantumsoft.qupathcloud.entities.Project}
+ * retrieving from the server.
+ *
```
Comment (poojavenkatram) [inline src/main/java/com/quantumsoft/qupathcloud/entities/Projects.java:24]: Projects contains a list of {@link com.quantumsoft.qupathcloud.entities.Project}
 retrieved from the server.

**Review by poojavenkatram — COMMENTED** [review]
Overall looks good, just a few small things.

**Comment by arvgord** [comment]
Thanks! I've changed all the places you mentioned.

**Review by poojavenkatram — APPROVED** [review]
Thanks for the changes :) LGTM.

**Pull request merged**